src/packages/apply-console-proxy/apply-console-proxy.test.ts

```diff
@@ -0,0 +1,20 @@
+import applyConsoleProxy from './';
+
+const options = {
+  log: true,
+  warn: true,
+  error: true,
+  clicks: true,
+  cookies: true,
+  localStorage: true,
+  sessionStorage: true,
+  clearOnReload: true,
+  ignoreScriptErrors: true,
+  plugins: []
+}
+
+describe('Apply Console Proxy', () => {
+  it('should return undefined', () => {
+    expect(applyConsoleProxy(options)).toBeUndefined();
+  });
+});
```

Comment:
Idk, but I feel that we should create more tests for this package in order to deliver more consistence. 
As it is already a function with a `() => void` signature, it is quite redundant to keep a test such as `should return undefined`. It is already specified in the function signature.

We should actually test the sideeffects caused by this function after `applyConsoleProxy(options)` is invoked.

Reply:
Maybe we can create a new issue to follow up this, since this might turn into a bigger task? @vitormargis @guilhermelimak

Reply:
This a part of conversation we've had in telegram, so i just committed this test to verify if the test is really necessary.

Reply:
Oki, we'll create a new issue to follow up on that =)